processor/transformprocessor/README.md

```diff
@@ -26,6 +26,8 @@ Supported functions:
 
 - `TraceID(bytes)` - `bytes` is a byte slice of exactly 16 bytes. The function returns a TraceID from `bytes`. e.g., `TraceID(0x00000000000000000000000000000000)`
 
+- `IsMatch(target, pattern)` - `target` is either a path expression to a telemetry field to retrieve or a literal string.  `pattern` is a regexp pattern. The function matches the target against the pattern, returning true if the match is successful and false otherwise.  If target is nil or not a string false is always returned. 
```

Comment:
I like sticking with our function syntax standardss, which says Factories don't need to be a verb but regular functions (maybe we should formalize a name for these functions, maybe transformers?) must be a verb.  Since IsMatch is a factory I like it not starting with a verb to avoid confusion with transformers.